src/feedlistformaction.cpp

```diff
@@ -215,7 +215,20 @@ bool FeedListFormAction::process_operation(Operation op,
 			std::shared_ptr<RssFeed> feed = v->get_ctrl()->get_feedcontainer()->get_feed(
 					pos);
 			if (feed) {
-				return open_feed_in_browser(feed);
+				const bool interactive = true;
+				return open_feed_in_browser(feed, interactive);
+			}
+		} else {
+			v->show_error(_("No feed selected!"));
+		}
+		break;
+	case OP_OPENINBROWSER_NONINTERACTIVE:
```

Comment:
Handlers for `OP_OPENINBROWSER` and `OP_OPENINBROWSER_NONINTERACTIVE` only differ by a single line (`const bool interactive`). Can they be extracted into a separate method that just takes `interactive` as an argument?

Same applies to all other dialogs that handle those operations.

If this requires too many changes, or intrusive changes that risk introducing bugs, I'd prefer to extract this work into a separate PR, to be merged after the 2.22 release.

Reply:
> Can they be extracted into a separate method that just takes interactive as an argument?

Yes, done.

> If this requires too many changes, or intrusive changes that risk introducing bugs, I'd prefer to extract this work into a separate PR, to be merged after the 2.22 release.

I think the changes are relatively save.
However, I'm fine if you want to drop them from this PR (and merge them after the 2.22 release).